### ISSUE TITLE
Facebook Pixels

### ISSUE THREAD

**Issue by Shorashim** [MarchingLizard/iwiorg#89]
Chicago Pixel:
```javascript
<!-- Facebook Conversion Code for Chicago Trips -->
<script>(function() {
var _fbq = window._fbq || (window._fbq = []);
if (!_fbq.loaded) {
var fbds = document.createElement('script');
fbds.async = true;
fbds.src = '//connect.facebook.net/en_US/fbds.js';
var s = document.getElementsByTagName('script')[0];
s.parentNode.insertBefore(fbds, s);
_fbq.loaded = true;
}
})();
window._fbq = window._fbq || [];
window._fbq.push(['track', '6020989615981', {'value':'0.00','currency':'USD'}]);
</script>
<noscript><img height="1" width="1" alt="" style="display:none" src="https://www.facebook.com/tr?ev=6020989615981&amp;cd[value]=0.00&amp;cd[currency]=USD&amp;noscript=1" /></noscript>
```

Washington DC Pixel:
```javascript
<!-- Facebook Conversion Code for Washington DC -->
<script>(function() {
var _fbq = window._fbq || (window._fbq = []);
if (!_fbq.loaded) {
var fbds = document.createElement('script');
fbds.async = true;
fbds.src = '//connect.facebook.net/en_US/fbds.js';
var s = document.getElementsByTagName('script')[0];
s.parentNode.insertBefore(fbds, s);
_fbq.loaded = true;
}
})();
window._fbq = window._fbq || [];
window._fbq.push(['track', '6021000873181', {'value':'0.00','currency':'USD'}]);
</script>
<noscript><img height="1" width="1" alt="" style="display:none" src="https://www.facebook.com/tr?ev=6021000873181&amp;cd[value]=0.00&amp;cd[currency]=USD&amp;noscript=1" /></noscript>
```

Pay it forward
```javascript
<!-- Facebook Conversion Code for Pay It Forward -->
<script>(function() {
var _fbq = window._fbq || (window._fbq = []);
if (!_fbq.loaded) {
var fbds = document.createElement('script');
fbds.async = true;
fbds.src = '//connect.facebook.net/en_US/fbds.js';
var s = document.getElementsByTagName('script')[0];
s.parentNode.insertBefore(fbds, s);
_fbq.loaded = true;
}
})();
window._fbq = window._fbq || [];
window._fbq.push(['track', '6028762005381', {'value':'0.00','currency':'USD'}]);
</script>
<noscript><img height="1" width="1" alt="" style="display:none" src="https://www.facebook.com/tr?ev=6028762005381&amp;cd[value]=0.00&amp;cd[currency]=USD&amp;noscript=1" /></noscript>
```

Homepage: 
```javascript
<!-- Facebook Conversion Code for Israelwithisraelis.com -->
<script>(function() {
var _fbq = window._fbq || (window._fbq = []);
if (!_fbq.loaded) {
var fbds = document.createElement('script');
fbds.async = true;
fbds.src = '//connect.facebook.net/en_US/fbds.js';
var s = document.getElementsByTagName('script')[0];
s.parentNode.insertBefore(fbds, s);
_fbq.loaded = true;
}
})();
window._fbq = window._fbq || [];
window._fbq.push(['track', '6017259126581', {'value':'0.00','currency':'USD'}]);
</script>
<noscript><img height="1" width="1" alt="" style="display:none" src="https://www.facebook.com/tr?ev=6017259126581&amp;cd[value]=0.00&amp;cd[currency]=USD&amp;noscript=1" /></noscript>
```

**Comment by IshaDakota** [comment]
which page is this supposed to be placed on?

**Comment by Shorashim** [comment]
Chicago and DC can link to http://www.israelwithisraelis.com/trip/options#community since there is not a way to link to that section of community trips individually, correct?

**Comment by Shorashim** [comment]
Pay It Forward should link to http://www.israelwithisraelis.com/pay-it-forward

**Comment by IshaDakota** [comment]
Can you try to research if you can... Not sure you can have 2 pixels on the
same page. If you can't I will look into it further.

**Comment by IshaDakota** [comment]
Also. On "pay it forward" do you want it on the landing page itself, or
only after someone submits the form?

**Comment by Shorashim** [comment]
If we can only link to the community page, you can just use the Chicago pixel. I changed the name and it can be the pixel for all community trips.

**Comment by Shorashim** [comment]
Didn't even know you could do that! I would say after someone submits the form because then we can better track conversions. That makes most sense, right?

**Comment by IshaDakota** [comment]
So, it seems to me that there is no problem putting multiple conversion pixels on one page, but the limited documentation on it suggests to me that maybe it's not the best practice. I did see a good bit of info on using a single pixel to track multiple ads... maybe this is what you want? Can you explore a little further, because my knowledge is a little limited.

For the pay it forward, what we can do is create a "thank you" page that results when someone submits the form and embed the pixel there. Could you give me some text for that page?

I think the safest most efficient way to do this is to hard code it (meaning that you won't be able to code it manually). Do you see this as a problem? Do you think they will change often?

**Comment by Shorashim** [comment]
Exactly - I think using a single pixel to track multiple ads is the best option for this. I have done this in the past and it has worked well.

I also think that the "thank you" page for Pay It Forward makes sense and will help us truly gage how well it is working. I cannot imagine having to change it often (or ever), so a hard code should be fine. Would I be able to change the text on the page as needed?

If so, text for page:
Thank you for referring your friends to the free trip of a lifetime. Pre-registration for December 2015 through August 2016 trips is currently ope here & it takes about one minute to apply: http://bit.ly/1gQqGQ7. Spread the word to your friends and family!

Winter applicants will be able to complete your application when registration opens in full on Tuesday, September 8th. Summer applicants will be able to continue their application when registration opens in February.

--
We should also do the same thing for the new "Tell Your Friends" page. This way, I can target applicants when registration opens through Facebook ads and accurately track their success. Should I create a new issue for the "Tell Your Friends" page?

**Comment by Shorashim** [comment]
Pixel codes were updated. I would like to place this on the homepage:


<script>
!function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
document,'script','//connect.facebook.net/en_US/fbevents.js');

fbq('init', '1766134580281502');
fbq('track', "PageView");</script>
<noscript><img height="1" width="1" style="display:none"
src="https://www.facebook.com/tr?id=1766134580281502&ev=PageView&noscript=1"
/></noscript>


--
Is there a way for me to add this? The instructions are here if I am not able to: https://www.facebook.com/business/help/218844828315224. We want to add ASAP, so we can utilize this for ads on Facebook. If there is anything I can do to help move forward next steps, please let me know.
